packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -240,9 +240,11 @@ export const TrackListItem = ({
               </Text>
             </View>
 
-            <View style={styles.downloadIndicator}>
-              <DownloadStatusIndicator trackId={track_id} size={18} />
-            </View>
+            {!isDeleted && (
```

Comment:
Can you apply the same logic in LineupTileStats?

Alternatively, move the check into DownloadStatusIndicator having it fetch the track or passing it in as a prop.

Reply:
I don't believe there is a way to see a deleted track in a lineup, or is there? We only show them in favorites and playlists so that they don't just dissapear when deleted

Reply:
Ah, that's a good point! This is fine then :)